Topics/Topic_2_b/src/main/java/com/bootcamp/topic2_b/blog/BlogList.java

```diff
@@ -0,0 +1,81 @@
+package com.bootcamp.topic2_b.blog;
+
+import java.util.Iterator;
+
+public class BlogList implements Iterable<Entry> {
```

Comment:
why BlogList? we need to model a blog, and make CRUD operations with the Entries. Need to re-do this topic dude :(